src/Common/Compat.targets

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Do we even need this targets file? I was kind of expecting we would just set the outputpath direction in the csprojs and that would be all we need.